README.md

```diff
@@ -17,6 +17,7 @@ Please also make sure all feature additions have a corresponding unit test.
 
 ## Release notes:
 
+- 7.5.1 Add routing key support, to support attribute based messaging. Fix hashing issue in dotnet core.
```

Comment:
Since this introduces a new feature (routing key), I believe the version should be `7.6.0` according to semantic verisoning.